src/tool/N3ME/MapMng.cpp

```diff
@@ -1429,19 +1430,19 @@ void CMapMng::MakeGameFiles(LPCTSTR lpszPathName, float fSize) {
     char szDrive[_MAX_DRIVE], szDir[_MAX_DIR], szFName[_MAX_FNAME], szExt[_MAX_EXT];
     _splitpath(lpszPathName, szDrive, szDir, szFName, szExt);
 
-    // ���� ����.
+    // Save file.
     HANDLE hFile = CreateFile(lpszPathName, GENERIC_WRITE, 0, NULL, CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, NULL);
     if (INVALID_HANDLE_VALUE == hFile) {
         MessageBox(::GetActiveWindow(), lpszPathName, "Fail to open map game data file for save, Pleas retry.", MB_OK);
         return;
     }
-    char  comment[80] = {"������ ���� ��� �ٺ�..^^"};
+    char  comment[80] = {"The person who opens this file is an idiot...^^"};
```

Comment:
I'm not sure if it's wise to translate this one, considering it's used directly in the files themselves. Though it's technically not checked, and still remains within the constraint, it does technically change the file formats unofficially. Whether or not that's a concern with this project, I don't know.

Reply:
@twostars has a point, however from my end this is fine, as this file has no logic nor used officially to check its content. In Korean it looks like random bytes, so at least in English we'll see the actual content lol